spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+      </emu-note>
+
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-pluralrules-prototype-object">
+    <h1>Properties of the Intl.PluralRules Prototype Object</h1>
+
+    <p>
+      The Intl.PluralRules prototype object is itself an ordinary object. <dfn>%PluralRulesPrototype%</dfn> is not an Intl.PluralRules instance and does not have an [[InitializedPluralRules]] internal slot or any of the other internal slots of Intl.PluralRules instance objects.
+    </p>
+    <p>
+      In the following descriptions of functions that are properties or [[Get]] attributes of properties of %PluralRulesPrototype%, the phrase "this PluralRules object" refers to the object that is the this  value for the invocation of the function; a *TypeError* exception is thrown if the this value is not an object or an object that does not have an [[InitializedPluralRules]] internal slot.
+    </p>
+
+    <emu-clause id="sec-intl.pluralrules.prototype.constructor">
+      <h1>Intl.PluralRules.prototype.constructor</h1>
+
+      <p>
+        The initial value of `Intl.PluralRules.prototype.constructor` is the intrinsic object %PluralRules%.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-intl.pluralrules.prototype-tostringtag">
+      <h1>Intl.PluralRules.prototype [ @@toStringTag ]</h1>
+
+      <p>
+        The initial value of the @@toStringTag property is the string value `"Object"`.
+      </p>
+      <p>
+        This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-intl.pluralrules.prototype.select">
+      <h1>Intl.PluralRules.prototype.select( _value_ )</h1>
+
+      <p>
+        When the `select` is called with an argument _value_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _pr_ be *this* value.
+        1. If Type(_pr_) is not Object, throw a *TypeError* exception.
+        1. If _pr_ does not have an [[InitializedPluralRules]] internal slot, throw a *TypeError* exception.
+        1. Let _n_ be ? ToNumber(_value_).
+        1. Return ? ResolvePlural(_pr_, _n_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-intl.pluralrules.prototype.resolvedoptions">
+      <h1>Intl.PluralRules.prototype.resolvedOptions ()</h1>
+
+      <p>
+        This function provides access to the locale and options computed during initialization of the object.
+      </p>
+
+      <emu-alg>
+        1. Let _pr_ be *this* value.
+        1. If Type(_pr_) is not Object, throw a *TypeError* exception.
+        1. If _pr_ does not have an [[InitializedPluralRules]] internal slot, throw a *TypeError* exception.
+        1. Let _options_ be ! ObjectCreate(%ObjectPrototype%).
+        1. For each row of <emu-xref href="#table-pluralrules-resolvedoptions-properties"></emu-xref>, except the header row, do
```

Comment:
@anba @littledan I took the liberty to not add the "in any order" from this algo step to avoid the same confusion that we have with the other algos. Are you guys ok with this?

Reply:
Seems good to me, I'm not sure why you'd put "in any order".

Reply:
Yes, I'm definitely okay with that. 😄

Reply:
If we don't want them in any order, should we sort the properties table alphabetically? It seems like we'll specify an an arbitrary order with it if I read it right.